homu/main.py

```diff
@@ -286,6 +289,25 @@ def parse_commands(body, username, repo_cfg, state, my_username, db, *, realtime
 
             state.save()
 
+        elif word.startswith('delegate='):
+            if try_only:
+                state.add_comment(':key: Insufficient privileges')
+                continue
+            state.delegate = word[len('delegate='):]
+            state.save()
+
+        elif word == 'delegate-':
+            state.delegate = ''
+            state.save()
+
+        elif word == 'delegate+':
+            if try_only:
+                state.add_comment(':key: Insufficient privileges')
+                continue
+            state.delegate = state.get_repo().pull_request(state.num).user
+            state.add_comment(':v: @{} can now approve this pull request'.format(state.delegate))
+            state.save()
```

Comment:
I generally try to place relevant code together, to reduce mistakes when moving code from here to there. In this aspect, `state.save()` might be better placed one line above, right after `state.delegate = ...`. Also, it would be better to insert a blank line after `state.save()`.